src/typechecker.erl

```diff
@@ -595,6 +595,8 @@ glb_ty(_Ty1, _Ty2, _A, _TEnv) -> {type(none), constraints:empty()}.
 %% * Replace built-in type synonyms
 %% * Flatten unions and merge overlapping types (e.g. ranges) in unions
 -spec normalize(type(), TEnv :: #tenv{}) -> type().
+normalize({type, _, term, []}, TEnv) ->
+    type(any);
```

Comment:
Now that I'm thinking about it: This could be placed in `expand_builtin_aliases`, since term() is listed as an alias for any() in http://erlang.org/doc/reference_manual/typespec.html. `expand_builtin_aliases` almost exactly follows this list.

Reply:
Yeah, that makes total sense. I've changed it. Thanks!